src/components/common/TmBalance.vue

```diff
@@ -190,6 +201,18 @@
               ><span>Send</span>
             </div>
           </div>
+
+          <div
```

Comment:
note: the fact that we need to duplicate this indicates that there is a need for a refactor. https://github.com/luniehq/lunie/issues/3942#issuecomment-623424830